### ISSUE TITLE
boundary problem

### ISSUE THREAD

**Issue by Lidabao77** [cselab/aphros#23]
Dr. Karnakov，in examples/205_multivof/clustering，how the boundary conditions are set. 
Is the following code setting the top and bottom boundaries to slipwall?
set string bc_path inline "
slipwall  {
  box 0 0 0 10
}
"
Thank you！

**Comment by pkarnakov** [comment]
Yes, this code sets top and bottom boundaries to slip walls. `box 0 0 0 10` means a box centered at `0 0 0` with a side of 20 (10 is half-size) in all directions. All cell faces that fall inside the box get their boundary conditions set to `slipwall`.
Here's some description of boundary conditions: https://cselab.github.io/aphros/doc/#boundary-conditions

**Comment by Lidabao77** [comment]
Thanks for the answer！

**Comment by gesx123** [comment]
你好 可以交流下吗，QQ2606612132
